zerver/tests/test_hooks.py

```diff
@@ -984,170 +1115,212 @@ class TaigaHookTests(WebhookTestCase):
     FIXTURE_DIR_NAME = 'taiga'
 
     def build_webhook_url(self):
+        # type: () -> text_type
         api_key = self.get_api_key(self.TEST_USER_EMAIL)
         return self.URL_TEMPLATE.format(stream=self.STREAM_NAME, api_key=api_key, topic=self.TOPIC)
```

Comment:
I think `URL_TEMPLATE` should be prefixed with a `u`, since otherwise it's difficult to see whether this function returns `text_type` or `str`.